packages/assets-controllers/src/Standards/NftStandards/ERC1155/ERC1155Standard.test.ts

```diff
@@ -48,4 +48,21 @@ describe('ERC1155Standard', () => {
       );
     expect(contractSupportsUri).toBe(true);
   });
+
+  it('should determine if contract supports token receiver interface correctly', async () => {
```

Comment:
This seems to be unrelated to the rest of the PR

Reply:
Its not related but the coverage went lower for some reason and the CI was failing, so i added this test just to update the coverage

Reply:
Oh I see, that's because we don't have coverage for our package exports, but a new package export was added.

We could address this by adding a snapshot test for package exports, similar to what we have already for the token price service module